.github/workflows/ci.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           zip -r fulfilment-lambdas.zip ./*
           popd
       - name: AWS Auth
-        uses: aws-actions/configure-aws-credentials@v2
+        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
fyi we are supposed to be moving away from this and to v4 of actions-riff-raff
https://github.com/guardian/actions-riff-raff?tab=readme-ov-file#migrating-from-v3-to-v4
might be a good one for next time someone's in tweaking this